go/page.templ

```diff
@@ -6,11 +6,35 @@ import (
 )
 
 // PageHeader provides a wrapper for the page header
-templ PageHeader() {
+templ PageHeader(p props.PageHeader) {
 	<div
-		class={ classes.PageHeader }
+		class={
+			templ.KV(classes.PageHeader, p.Class == ""),
+			templ.KV(p.Class, p.Class != ""),
+		}
+		{ p.Attrs... }
 	>
-		{ children... }
+		<div class={ classes.PageTitleWrapper }>
+			if p.Breadcrumbs != nil {
+				@PageBreadcrumbs() {
+					for _, i := range p.Breadcrumbs {
+						<li>
+							if i.URL != "" {
+								<a href={ i.URL }>{ i.Name }</a>
```

Comment:
We'll need to support Attributes here as well to be able to support extended features.